src/main/java/org/jboss/tools/intellij/analytics/CliConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.tools.intellij.analytics;
+
+public class CliConfig {
+    public static final String CLI_AUTH = "yes \"\" | binarypath/crda auth";
+    public static final String CLI_ANALYSE = "crda analyse filepath -j -c";
+    public static final String CLI_CONFIG_FILE = System.getProperty("user.home")+".crda/config.yaml";
```

Comment:
Use Paths.get to concat path, it will be platform agnostic
https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html#get(java.lang.String,%20java.lang.String...)